runtime/src/main/java/org/corfudb/util/retry/ExponentialBackoffRetry.java

```diff
@@ -83,4 +88,8 @@ public void nextWait() {
         Sleep.MILLISECONDS.sleepUninterruptibly(sleepTime);
     }
 
+    public void reset() {
```

Comment:
Why not reseting the retryCounter in the reset method?

Reply:
Good catch. Done.